api/src/main/java/uk/gov/dvsa/mot/persist/jdbc/VehicleReadDaoJdbc.java

```diff
@@ -1,5 +1,6 @@
 package uk.gov.dvsa.mot.persist.jdbc;
 
+import com.google.common.base.Strings;
```

Comment:
FIY @raffcecco uses `com.amazonaws.util.StringUtils` for nullOrEmpty purposes. We could agree on one. ;> (given we have a proper library in all the projects)

Reply:
I mentioned that in a review on Raff's code. Personally, I think we should use Google's common library and not Amazon's library, which creates an explicit dependency on AWS code.